client-sdk/go/testing/testing.go

```diff
@@ -21,15 +22,20 @@ type TestKey struct {
 
 	// EthAddress is the corresponding Ethereum address if the key is secp256k1.
 	EthAddress [20]byte
+
+	// UnsafeBytes stores the Signer's private keye.
+	UnsafeBytes []byte
 }
 
 func newEd25519TestKey(seed string) TestKey {
 	signer := ed25519.WrapSigner(memorySigner.NewTestSigner(seed))
 	sigspec := types.NewSignatureAddressSpecEd25519(signer.Public().(ed25519.PublicKey))
+	s := sha512.Sum512_256([]byte(seed))
 	return TestKey{
-		Signer:  signer,
-		Address: types.NewAddress(sigspec),
-		SigSpec: sigspec,
+		Signer:      signer,
+		Address:     types.NewAddress(sigspec),
+		SigSpec:     sigspec,
+		UnsafeBytes: ed25519voi.NewKeyFromSeed(s[:]),
```

Comment:
Could you use the `UnsafeBytes()` method available on the signer returned by `memorySigner.NewTestSigner`? You can do something like:
```rust
ms := memorySigner.NewTestSigner(seed)

// ...

    UnsafeBytes: ms.(signature.UnsafeSigner).UnsafeBytes()
```